tests/test_piwik.py

```diff
@@ -14,6 +14,8 @@
 
 # Doesn't check that the action was actually tracked, just that it doesn't crash.
 # You can manually check online at TRACKER_URL that the action was indeed tracked.
+# This test never exits. It needs no be manually exited (ctrl + c).
```

Comment:
Even if the tracker shouldn't stop, this feature might be useful for test. 
To do so, we can add this code and call `stop_test(tracker)`:
```
    def stop_test(tracker):
        def stop_timer(tracker):
            return
        tracker.stop_timer = types.MethodType(stop_timer, PiwikTracker)
```